src/map/mob.cpp

```diff
@@ -4187,188 +4290,663 @@ static bool mob_parse_dbrow(char** str)
+		}
+		else
+			status->mode = static_cast<enum e_mode>(status->mode & ~MD_CHANGETARGET_MELEE);
+
+		if (this->nodeExists(modeNode, "ChangeTargetChase"))
+		{
+			if (this->asBool(modeNode, "ChangeTargetChase", mode))
+				status->mode = mode ? static_cast<enum e_mode>(status->mode | MD_CHANGETARGET_CHASE) : static_cast<enum e_mode>(status->mode & ~MD_CHANGETARGET_CHASE);
+		}
+		else
+			status->mode = static_cast<enum e_mode>(status->mode & ~MD_CHANGETARGET_CHASE);
+
+		if (this->nodeExists(modeNode, "TargetWeak"))
+		{
+			if (this->asBool(modeNode, "TargetWeak", mode))
+				status->mode = mode ? static_cast<enum e_mode>(status->mode | MD_TARGETWEAK) : static_cast<enum e_mode>(status->mode & ~MD_TARGETWEAK);
+		}
+		else
+			status->mode = static_cast<enum e_mode>(status->mode & ~MD_TARGETWEAK);
+
+		if (this->nodeExists(modeNode, "RandomTarget"))
+		{
+			if (this->asBool(modeNode, "RandomTarget", mode))
+				status->mode = mode ? static_cast<enum e_mode>(status->mode | MD_RANDOMTARGET) : static_cast<enum e_mode>(status->mode & ~MD_RANDOMTARGET);
+		}
+		else
+			status->mode = static_cast<enum e_mode>(status->mode & ~MD_RANDOMTARGET);
+
+		if (this->nodeExists(modeNode, "IgnoreMeleeDamage"))
+		{
+			if (this->asBool(modeNode, "IgnoreMeleeDamage", mode))
+				status->mode = mode ? static_cast<enum e_mode>(status->mode | MD_IGNOREMELEE) : static_cast<enum e_mode>(status->mode & ~MD_IGNOREMELEE);
+		}
+		else
+			status->mode = static_cast<enum e_mode>(status->mode & ~MD_IGNOREMELEE);
+
+		if (this->nodeExists(modeNode, "IgnoreMagicDamage"))
+		{
+			if (this->asBool(modeNode, "IgnoreMagicDamage", mode))
+				status->mode = mode ? static_cast<enum e_mode>(status->mode | MD_IGNOREMAGIC) : static_cast<enum e_mode>(status->mode & ~MD_IGNOREMAGIC);
+		}
+		else
+			status->mode = static_cast<enum e_mode>(status->mode & ~MD_IGNOREMAGIC);
+
+		if (this->nodeExists(modeNode, "IgnoreRangedDamage"))
+		{
+			if (this->asBool(modeNode, "IgnoreRangedDamage", mode))
+				status->mode = mode ? static_cast<enum e_mode>(status->mode | MD_IGNORERANGED) : static_cast<enum e_mode>(status->mode & ~MD_IGNORERANGED);
+		}
+		else
+			status->mode = static_cast<enum e_mode>(status->mode & ~MD_IGNORERANGED);
+
+		if (this->nodeExists(modeNode, "MVP"))
+		{
+			if (this->asBool(modeNode, "MVP", mode))
+				status->mode = mode ? static_cast<enum e_mode>(status->mode | MD_MVP) : static_cast<enum e_mode>(status->mode & ~MD_MVP);
+		}
+		else
+			status->mode = static_cast<enum e_mode>(status->mode & ~MD_MVP);
+
+		if (this->nodeExists(modeNode, "IgnoreMiscSkills"))
+		{
+			if (this->asBool(modeNode, "IgnoreMiscSkills", mode))
+				status->mode = mode ? static_cast<enum e_mode>(status->mode | MD_IGNOREMISC) : static_cast<enum e_mode>(status->mode & ~MD_IGNOREMISC);
+		}
+		else
+			status->mode = static_cast<enum e_mode>(status->mode & ~MD_IGNOREMISC);
+
+		if (this->nodeExists(modeNode, "KockbackImmune"))
```

Comment:
@lukasrmattos Is it possible to rename this to `KnockbackImmune`?

Reply:
Done